README.md

```diff
@@ -1,45 +1,79 @@
-![Sylius](https://dl.dropboxusercontent.com/u/46579820/sylius-logo.jpg)
+![Sylius](http://demo.sylius.org/assets/shop/img/logo.png)
 
 [![Gitter chat](https://badges.gitter.im/Sylius/Sylius.png)](https://gitter.im/Sylius/Sylius)
 [![License](https://img.shields.io/packagist/l/Sylius/Sylius.svg)](https://packagist.org/packages/sylius/sylius)
+[![Version](https://img.shields.io/packagist/v/Sylius/Sylius.svg)](https://packagist.org/packages/sylius/sylius)
+[![Build status on Linux](https://img.shields.io/travis/Sylius/Sylius/master.svg)](http://travis-ci.org/Sylius/Sylius)
+[![Scrutinizer Quality Score](https://img.shields.io/scrutinizer/g/Sylius/Sylius.svg)](https://scrutinizer-ci.com/g/Sylius/Sylius/)
+[![Dependency Status](https://www.versioneye.com/php/sylius:sylius/badge.svg)](https://www.versioneye.com/php/sylius:sylius)
+[![Total Downloads](https://poser.pugx.org/sylius/sylius/downloads)](https://packagist.org/packages/sylius/sylius)
 
-Sylius is an open source e-commerce solution for **PHP**, based on the [**Symfony2**](http://symfony.com) framework.
+**This is Sylius Standard Edition repository for starting new projects.**
 
-Ultimate goal of the project is to create a webshop engine, which is user-friendly, *loved* by developers and has a helpful community.
+Sylius is the first decoupled eCommerce framework based on [**Symfony**](http://symfony.com) and [**Doctrine**](http://doctrine-project.org). 
+The highest quality of code, strong testing culture, built-in Agile (BDD) workflow and exceptional flexibility make it the best solution for application tailored to your business requirements. 
+Enjoy being an eCommerce Developer again!
 
-Sylius is constructed from fully decoupled components (bundles in Symfony2 glossary), which means that every feature (products catalog, shipping engine, promotions system...) can be used in any other application. 
+We're using full-stack Behavior-Driven-Development, with [phpspec](http://phpspec.net) and [Behat](http://behat.org)
 
-We're using full-stack BDD methodology, with [phpspec](http://phpspec.net) and [Behat](http://behat.org).
+Documentation
+-------------
 
-This package contains the Sylius Standard Edition, which serves as a foundation for your custom apps.
+Documentation is available at [docs.sylius.org](http://docs.sylius.org).
 
 Installation
 ------------
 
-``` bash
+```bash
 $ wget http://getcomposer.org/composer.phar
-$ php composer.phar create-project sylius/sylius-standard path/to/install
-$ cd path/to/install
+$ php composer.phar create-project sylius/sylius-standard app
```

Comment:
`app` can be confusing with the `app` folder with Symfony confugiration, maybe `project` will be better?